tasty-plutus/src/Test/Tasty/Plutus/Script/Property.hs

```diff
@@ -210,6 +213,75 @@ scriptPropertyPass ::
   WithScript p ()
 scriptPropertyPass = mkScriptPropertyWith OutcomeAlwaysPass
 
+{- | Given a 'Generator' containig a way to generate a seed and function
+ to create 'TestItems' as well as a function to create "TestScript' from the seed,
```

Comment:
Add a comma between 'TestItems' and 'as well as':

to create 'TestItems', as well as ...